Various improvements

arthomnix opened the pull request:
* Update to 1.19 release, update FAPI, FLoader and Quilt Mappings
* Fix descriptions sometimes containing HTML tags
* Add tooltips that show the full description of packs
* Use a try-with-resources statement in VTDMod.getCategories to make sure the CloseableHttpClient gets closed

arthomnix committed:
Various improvements
Update to 1.19 release, update FAPI, FLoader and Quilt Mappings
Fix descriptions sometimes containing HTML tags
Add tooltips that show the full description of packs

IotaBread reviewed (approved):
Thanks for the contribution! Looks good, just a single thing - could you add back the removed new lines at the end of the files? If not, I can do it myself, no problem there

arthomnix committed:
Add trailing newlines

arthomnix commented:
I've also added icons, and deactivating the button for the currently selected tab, would probably be easiest if I added them to this PR

![image](https://user-images.githubusercontent.com/35371030/173679419-339ea94b-1cef-49c8-9dc9-75600bbeec2d.png)

(also forgot to mention in the original comment that it now uses 1.19 vanilla tweaks packs)
